.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```

Comment:
**Nitpick**
Please, use [editorConfig](http://editorconfig.org/) to remove `EOF`